app/models/admin_user.rb

```diff
@@ -0,0 +1,6 @@
+class AdminUser < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
+end
```

Comment:
- [ ] This devise configuration should be in the User `(models/user.rb)` model. There's no apparent need to create an admin_user model here. Kindly move this config to the User model.